mithril-test-lab/mithril-end-to-end/src/mithril/infrastructure.rs

```diff
@@ -38,27 +40,34 @@ pub struct MithrilInfrastructure {
     signers: Vec<Signer>,
     relay_aggregators: Vec<RelayAggregator>,
     relay_signers: Vec<RelaySigner>,
+    relay_passives: Vec<RelayPassive>,
     cardano_chain_observer: Arc<dyn ChainObserver>,
     run_only_mode: bool,
     is_signing_cardano_transactions: bool,
 }
 impl MithrilInfrastructure {
     pub async fn start(config: &MithrilInfrastructureConfig) -> StdResult<Self> {
+        let chain_observer_type = "pallas";
```

Comment:
It's not a new thing from this PR, but would it be useful and possible to use directly `ChainObserverType` enum instead of a `&str`?